pkg/openmetadata-connector-core/connector_service.go

```diff
@@ -155,7 +161,12 @@ func (s *OpenMetadataAPIService) DeleteAsset(ctx context.Context, xRequestDataca
 		s.initialized = s.PrepareOpenMetadataForFybrik()
 	}
 
-	c := s.getOpenMetadataClient()
+	c, err := s.getOpenMetadataClient(ctx)
+	if err != nil {
+		s.logger.Error().Err(err).Msg(CannotLoginToOM)
+		return api.Response(http.StatusUnauthorized, nil), errors.New(CannotLoginToOM)
```

Comment:
I would suggest to separate several errors, e.g. server error, wrong URL, wrong request + unauthorized.

Reply:
I don't know how to differentiate between these different problems.
Hopefully, `err` contains useful information.